com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -86,6 +86,12 @@ public NetworkPrefabHandler PrefabHandler
         private bool m_ShuttingDown;
         private bool m_StopProcessingMessages;
 
+        // <summary>
```

Comment:
@ShadauxCat 's idea of also having a reason param in OnClientDisconnectCallback would be great too.

Reply:
I guess you can always just access DisconnectReason from there as well statically...
Switching the above as just nitpicking

Reply:
We can't change `OnClientDisconnectCallback`, that would be a breaking change.